lighthouse-core/formatters/partials/url-list.html

```diff
@@ -15,7 +15,10 @@
     <summary>URLs</summary>
     {{#each this}}
       <div class="http-resource">
-        <span class="http-resource__url">{{this.url}}</span><span class="http-resource__protocol">({{this.protocol}})</span>
+        <span class="http-resource__url">{{this.url}}</span>
```

Comment:
Shouldn't overload `url` here like this. Maybe just add a place for a more general text string that is styled the same?